sdks/python/apache_beam/io/textio.py

```diff
@@ -86,7 +90,9 @@ def __init__(self,
                coder,
                buffer_size=DEFAULT_READ_BUFFER_SIZE,
                validate=True,
-               skip_header_lines=0):
+               skip_header_lines=0,
+               header_matcher_predicate=None,
```

Comment:
<!--new_thread; commit:70ade9d4fe6a8e31ff0be4d39b731510c93b4bf9; resolved:0-->
Please document what 'header_matcher_predicate' and 'header_processor' should be since they are not documented anywhere else.